EpamTests/StepDefinitions/CarouselSteps.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using EpamTests.Pages;
+using EpamTests.Tests;
+using TechTalk.SpecFlow;
+
+namespace EpamTests.StepDefinitions
+{
+    [Binding]
+    public class CarouselSteps : BaseTest
+    {
+        private string? expectedTitle;
+        private string? actualTitle;
```

Comment:
dont use shared fields. the methods should be independent between each other. so just in the method get actual and expected and compare